bluebottle/test/utils.py

```diff
@@ -49,9 +49,12 @@ def css_dict(style):
 class InitProjectDataMixin(object):
 
     def init_projects(self):
+        from django.core import management
         """
         Set up some basic models needed for project creation.
         """
+        management.call_command('loaddata', 'project_data.json', verbosity=0)
```

Comment:
I think it is faking those migrations then because without the mangement command the project phases don't get loaded in the tests.